src/app/products/products-routing.module.ts

```diff
@@ -239,7 +254,7 @@ const routes: Routes = [
               path: '',
               component: FixedDepositProductsComponent,
               resolve: {
-                    fixedDepositProducts: FixedDepositProductsResolver
+                fixedDepositProducts: FixedDepositProductsResolver
```

Comment:
Remove all these corrections to indentation from this PR. This should not be a part of this.